haystack/nodes/__init__.py

```diff
@@ -49,11 +49,17 @@
     Text2SparqlRetriever,
     TableTextRetriever,
     MultiModalRetriever,
+    WebRetriever,
 )
 
 from haystack.nodes.sampler import BaseSampler, TopPSampler
+from haystack.nodes.search_engine import SearchEngine, WebSearch
```

Comment:
Technically `SearchEngine` is not a node, so I would not expose it... arguable though. Do you have some good reason to expose it (typing, subclassing...)?

Reply:
Ah... you are probably right and I don't have a deep understatement of the new web nodes...

I only saw `WebSearch` imported several times in the [docs](https://docs.haystack.deepset.ai/docs/search_engine), but at this point, I think there is an error in the docs...

If you agree, I can not expose `SearchEngine`, but only `WebSearch`.
And then correct the docs...

Reply:
It doesn't have to be exposed for now, as devrel is concerned about additional complexity. Super users will find it and use it. More important is the subclass registration mechanism Sara introduced in prompt node invocation layers. We can use it here as well (WebSeearch and SearchEngine).

Reply:
ok.
I don't expose `SearchEngine` now...